setup.py

```diff
@@ -0,0 +1,18 @@
+import setuptools
+
+setuptools.setup(
+    name='executor-clip',
+    version='2.0',
+    author='Jina Dev Team',
+    author_email='dev-team@jina.ai',
+    description='Executors that encode images',
+    url='https://github.com/jina-ai/executor-clip-image',
+    classifiers=[
+        'Programming Language :: Python :: 3',
+        'Operating System :: OS Independent',
+    ],
+    py_modules=['jinahub.encoder.image.clip'],
```

Comment:
is this correct?

Reply:
we can have that path for now and change it later in case we see a reason to change the structure